src/test/java/com/home/eshop/dao/BikesCacheTest.java

```diff
@@ -1,71 +1,96 @@
 package com.home.eshop.dao;
 
 import com.home.eshop.model.Bike;
-import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
 
 import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.times;
 
 class BikesCacheTest {
-    String path = getClass().getClassLoader().getResource("BikeBaseTestOne.txt").getPath();
 
-    Dao bikesCache = new BikesCache(path);
+    @Mock
+    private BikesDao bikesDao;
+    private final BikesCache bikesCache;
+    @Spy
+    Bike bikeOne = new Bike("One", 1, 1, 1);
+    Bike bikeUpdate = new Bike("Changed", 1, 1, 1);
+    Bike bikeTwo = new Bike("Two", 2, 2, 2);
 
-    Bike bikeOne = new Bike("testOne", 1, 1, 1);
-    Bike bikeTwo = new Bike("testTwo", 2, 2, 2);
-    Bike bikeOneChange = new Bike("testOneChange", 3, 3, 1);
-
-    Bike bikeActual;
+    public BikesCacheTest() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
it's  better to use annotation (mentioned it above)

Reply:
Done